docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -137,6 +137,22 @@ This prop is available on all fields and pickers.
 
 {{"demo": "CustomFieldFormat.js"}}
 
+### Custom field placeholder
+
+When a section is empty, the fields displays its placeholder instead of an empty value.
+For example, if you did not fill any value for the `year` section, the field will render the year placeholder.
+
+These placeholders are based on your current component localization, not on your date localization.
+
+{{"demo": "FieldPlaceholder.js"}}
+
+For more information on how to define your component localization, check out the [Translated components](/x/react-date-pickers/localization/) page.
+
+:::warning
+Not all locale are translating the placeholders.
+Some are keeping the English letters because it is what is usually done for the dates in those languages.
```

Comment:
Placeholders translations depend on locale.
Some locales might keep using English placeholders, because that format is commonly used in a given locale.